doc/managing_pachyderm/migrations.md

```diff
@@ -0,0 +1,291 @@
+    - [6. Restore the new 1.Y Pachyderm cluster from your backup](#restore-the-new-1-y-pachyderm-cluster-from-your-backup)
+    - [7. Load transactional data from checkpoint into new cluster](#load-transactional-data-from-checkpoint-into-new-cluster)
+    - [8. Disable the old cluster](#disable-the-old-cluster)
+    - [9. Reconfigure new cluster as necessary](#reconfigure-new-cluster-as-necessary)
+
+## Introduction
+As new versions of Pachyderm are released, you may need to update your cluster to get access to bug fixes and new features. 
+These updates fall into two categories, upgrades and migrations.
+
+An upgrade is moving between point releases within the same major release, 
+like 1.7.2 to 1.7.3.
+Upgrades are typically a simple process that require little to no downtime.
+They're covered in a [separate document](./upgrades.html).
+
+Migrations involve moving between major releases, 
+like 1.8.6 to 1.9.0.
+Migration is covered in this document. 
+
+In general, 
+Pachyderm stores all of its state in two places: 
+`etcd` 
+(which in turn stores its state in one or more persistent volumes,
+which were created when the Pachyderm cluster was deployed) 
+and an object store bucket 
+(something like AWS S3, MinIO, or Azure Blob Storage).
+
+In a migration, 
+the data structures stored in those locations need to be read, transformed, and rewritten, 
+so the process involves:
+
+1. bringing up a new Pachyderm cluster adjacent to the old pachyderm cluster
+1. exporting the old Pachdyerm cluster's repos, pipelines, and input commits
+1. importing the old cluster's repos, commits, and pipelines into the new
+   cluster, and then re-running all of the old cluster's pipelines from scratch
+   once.
+
+*You must perform a migration to move between major releases*,
+such as 1.8.7 to 1.9.0.
+
+Whether you're doing an upgrade or migration, it is recommended you [backup Pachyderm](./backup_restore.html#general-backup-procedure) prior. 
+That will guarantee you can restore your cluster to its previous, good state. 
+
+## Note about 1.7 to 1.8 migrations
+
+In Pachyderm 1.8,
+we rearchitected core parts of the platform to [improve speed and scalability](http://www.pachyderm.io/2018/11/15/performance-improvements.html). 
+Migrating from 1.7.x to 1.8.x using the procedure below can a fairly lengthy process. 
+If your requirements fit, it may be easier to create a new 1.8 or greater cluster and reload your latest source data into your input repositories.
+
+You may wish to keep your original 1.7 cluster around in a suspended state, reactivating it in case you need access to that provenance data.
+
+## General migration procedure
+
+### Before you start: backups
+
+Please refer to [the documentation on backing up your cluster](./backup_restore.html#general-backup-procedure).
+
+### Migration steps
+#### 1. Pause all pipeline and data loading operations
```

Comment:
Isn't there a ton of overlap here with the backup doc? Would it be easier/simpler to just say "follow the backup procedures doc to pause your pipelines, create a backups using pachctl extract"? Or is there something I'm missing?

Reply:
Remember that we're telling them to do this *alongside* the original cluster.  There is some duplication, but I'm going to assume that they want a backup *and* they want to do this separate operation, because backups are going to go into a specific backup area vs. the operational area they use here.